diff --git a/Makefile b/Makefile
index a467b7f..4b16a2d 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,3 @@
-FREEBSD-GE-10 = $(shell test `uname` = FreeBSD -a `uname -r | cut -d. -f1` -ge 10 && echo YES)
-CC = $(if $(FREEBSD-GE-10),clang,gcc)
-
 NAME = flisp
 SRCS = $(NAME).c builtins.c string.c equalhash.c table.c iostream.c
 OBJS = $(SRCS:%.c=%.o)
@@ -17,7 +14,7 @@ LIBS = $(LIBFILES) -lm
 DEBUGFLAGS = -g -DDEBUG $(FLAGS)
 SHIPFLAGS = -O2 -DNDEBUG $(FLAGS)
 
-default: release test
+default: release
 
 test:
 	cd tests && ../flisp unittest.lsp
